packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -145,6 +145,7 @@
     }
 
     &.pageless {
+      // display: flex;
```

Comment:
nit: We can remove this comment if it is not needed.

Reply:
good catch. fixed: https://github.com/cerner/terra-core/pull/3872/commits/2b488819c07215061791907e4d16fce30a0cf270#diff-639527009c6ffee66446a77ca5dec0c77f73dec484857a448936050b3f7a2e19L148